### PR TITLE
Pass the index url to both the 'installed' and 'buildout' checkers.

### PR DESCRIPTION
**Pull request by janjaapdriessen**

### CODE REVIEW

File: src/z3c/checkversions/main.py [hunk 1]
```diff
@@ -82,17 +82,15 @@ def main():
...
         from . import installed
-        checker = installed.Checker(blacklist=options.blacklist,
-                                    incremental=options.incremental,
-                                    verbose=options.verbose)
+        factory = installed.Checker
 
+    checker = factory(blacklist=options.blacklist,
+                      incremental=options.incremental,
+                      verbose=options.verbose,
+                      **kw)
```
Comment (mgedmin) [inline src/z3c/checkversions/main.py:93]: You're no longer passing `filename` to `buildut.Checker`?  How do things work, given
https://github.com/zopefoundation/z3c.checkversions/blob/be59ba7b131b0773e10018abdccf6a40a545659e/src/z3c/checkversions/buildout.py#L29-L33

Is the code untested?

Comment (janjaapdriessen) [reply src/z3c/checkversions/main.py]: Good catch @mgedmin, I fixed passing the filename and added tests for the main() function in b77bcf803fae2bad387ce2ba5698f20e6cec7b04

**Comment by janjaapdriessen** [comment]
@mgedmin are there any more metadata changes required before making a release?
I see some discussions about packaging in other zopefoundation repos and wonder how I can contribute. 

If there are no changes required, would you add me on pypi (janjaapdriessen) for this package so I can make a release?

**Comment by mgedmin** [comment]
> @mgedmin are there any more metadata changes required before making a release?

I'm not sure?  When making a release I tend to run a couple of tools (check-manifest, check-python-versions), unless I'm feeling lazy; look at git shortlog since the last release and compare it with git diff of CHANGES.rst (to make sure every user-visible change is mentioned); then run zest.releaser's `fullrelease`.

I have a couple of [git aliases](https://github.com/mgedmin/dotfiles/blob/c24e24b47290f4db6757cab5bf805121ca4d6fda/gitconfig#L126-L141) (`git unreleased`, `git unreleased-changelog`) that help me do the above.

There are some other bits of metadata that might be nice to keep up-to-date, if you notice they're outdated:

- `url` in setup.py should point to the GitHub repository, and not some old PyPI page
- as per https://github.com/zopefoundation/meta/issues/17 we might need to start adding `python_requires`
- nothing else comes to mind?

> If there are no changes required, would you add me on pypi (janjaapdriessen) for this package so I can make a release?

Done!

**Pull request merged**